### DISCUSSION TITLE
Move "Reset Filters" Button to Navigation Popup

### DISCUSSION THREAD

**Discussion by paur94** [KevinVandy/material-react-table#1191] [Q&A]
Hi,

Is it possible to move the "Reset Filters" button into the navigation popup? Also, can I add a 'Clear sort' button in that popup?
<img width="813" alt="image" src="https://github.com/user-attachments/assets/140f49b3-7840-4ce2-b144-13c2a18bc381">

**Comment by paur94** [comment]
@KevinVandy please look into this

**Reply by KevinVandy** [reply]
As I've told many people before, if you want a custom popup there, just build your own. It's actually not that hard. You can start by copying the source code for that popup and forking it however you want.
